### PR TITLE
Replace Wiki Page: Calibrating_Audio_Levels

### PR DESCRIPTION
**Pull request by knobby4444**
Copied the wiki page into md format.  Submitting just this one to see how it goes.   Next PR can include all the other open issues for wiki page transfer together if all goes well.

### CODE REVIEW

File: docs/adv-topics/autopatch.md
Comment (Allan-N) [inline docs/adv-topics/autopatch.md:1]: Can you remove the changes to autopatch.md and sip-phone.md from this branch?

File: docs/adv-topics/audio-level.md
Comment (Allan-N) [inline docs/adv-topics/audio-level.md:1]: Please have a look at the [recently updated] "Setting Audio Levels" section in ".../docs/adv-topics/usbinterfaces.md 
docs/adv-topics/usbinterfaces.md".

**Review by Allan-N — DISMISSED** [review]
When adding new pages you also need to update `.../mkdocs.yml` file and `.../docs/adv-topics/index.md`.  That helps with the topic lists, index, etc.

File: docs/adv-topics/audio-level.md
Comment (knobby4444) [reply docs/adv-topics/audio-level.md:1]: Ah, do we still want to include the stuff I copied from the wiki?  It feels like the wiki page porcdess using a service monitor is something most people won't be doing.  And maybe that people with access to a service monitor will already know how to check deviation, etc.  Which parts, if any, of the wiki page do you want added to the usbinterfaces.md?

Comment (Allan-N) [reply docs/adv-topics/audio-level.md:1]: Also, tagging @KB4MDD and @davidgsd for their thoughts.

File: docs/adv-topics/autopatch.md
Comment (knobby4444) [reply docs/adv-topics/autopatch.md:1]: I tried but not sure how to do this.  I'm not sure that after being retired I'm going to be able to use git again.  It's always been something that I've had to use some, but not enough to get good at it.

Comment (Allan-N) [reply docs/adv-topics/autopatch.md:1]: Google'ing around suggests :

```
git restore --source origin/main filename
```

and if that pulls back the original file you may need to add/commit/push to update the branch.

File: docs/adv-topics/audio-level.md
Comment (Allan-N) [reply docs/adv-topics/audio-level.md:1]: The changes, as-is, need some work.  I found many places where list items are really really really long lines.  In these cases, a markdown list (ordered or unordered) would work MUCH better.

Please review the [Markdown](https://daringfireball.net/projects/markdown/) formatting options.

**Review by jxmx — APPROVED** [review]

**Pull request merged**